cmd/brew-services.rb

```diff
@@ -155,9 +160,18 @@ def list
       puts format("#{Tty.white}%-#{longest_name}.#{longest_name}s %-7.7s %-#{longest_user}.#{longest_user}s %s#{Tty.reset}",
                   "Name", "Status", "User", "Plist")
       formulae.each do |formula|
+        status = case formula[:status]
+                 when :started
```

Comment:
Indent `when :started` two-chars after `status`. Sorry for pedantry, a pet hate 😉